### DISCUSSION TITLE
Error in invocation of downloads.download(downloads.DownloadOptions options, optional function callback): Error at parameter 'options': Unexpected property: 'incognito'.

### DISCUSSION THREAD

**Discussion by trewwert22** [aclap-dev/video-downloadhelper#307] [Help and Q&A]
### Your question

Hello,

I was trying to download a video off of a *spicy* website but whenever I go to download the video, it brings up the same error that is show in the title. I'm completely lost on what to do about it so I am hoping for some guidance, thank you! I've looked into previous discussions and I've never used a coapp before so if that is a likely fix please advise. Thanks again!

### Addon & CoApp version and details

version: 8.2.2.8
channel: stable
build date: 2024-03-18
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: unset
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/122.0.0.0 Safari/537.36
coapp: no.

**Comment by GrampaWildWilly** [comment]
It would help if you posted a URL, no matter how spicy.  You could post the URL inside a text attachment in order to make it a bit more discreet.  Also, you are not using the CoApp.  That could be an issue.  The content might require the CoApp to download it.  But I can't determine that without a URL.

**Reply by trewwert22** [reply]
Those are all good points, here is the link: https://erothots.co/video/mjtkijwqp/blue-bikini-annamolli/

I should also mention that I did download videos on this site a couple days ago and it worked fine, I'm wondering why it isn't working now

**Reply by GrampaWildWilly** [reply]
I'm running Windows 11 Pro 64-bit version 23H2 build 22631.3296, Firefox 123.0.1 64-bit, licensed VDH 8.2.2.8a1 beta, CoApp 2.0.19.  I do not work for VDH.  I'm just another user.

So I paid a visit to Miss Blue Bikini:

![#01](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/553f0115-4ab2-4314-8c42-f740da8579db)

As you can see, she gave me only one way to download her.  So I did.  Here's how she arrived on my computer:

![#02](https://github.com/aclap-dev/video-downloadhelper/assets/75588964/39da97ca-16e5-4735-a335-c011c37f58c7)

I thought the silent treatment might be a bit rude.  But playing her in her web page confirmed that she indeed has nothing to say, none of the usual moaning & sighing, not even music to accompany her antics.  I would show you a clip from VLC of her moves but we need to keep this more or less SFW.

She was perfectly happy to let me go download on her with the CoApp, & a second time with the browser.  Apparently, she's not too picky about such things.

The major difference between our setups is the browser.  Perhaps the availability of more advanced beta versions of VDH for Firefox allowed me to convince Miss Blue Bikini to play with me.  This would not be the first time something worked better on Firefox than on the other browsers.  Maybe Firefox is just sexier.  It's certainly not about the CoApp seeing as she came when I called on her without the CoApp.

**Comment by paulrouget** [comment]
I broke that. See my reply here: https://github.com/aclap-dev/video-downloadhelper/discussions/308#discussioncomment-8834832
